blocks/footer/footer.css

```diff
@@ -1,14 +1,451 @@
+        margin: 0;
+    }
+
+    &:first-child  { /* Footer Top : Logo */
+      border-right: 1px solid var(--primary-blue-80);
+      border-left: 0;
+    }
+
+    &:nth-child(2)  { /* Footer Top : Brands */
+      padding: 0;
+    }
+    
+    &:last-child { /* Footer Top : Social Links */
+    
+      & ul {
+        display:flex;
+        flex-wrap: nowrap;
+        justify-content: flex-end;
+    
+        & li {
+          padding: 0 1.5rem 0 0;
+
+          & span.icon {
+            width: 50px;
+            height: 50px;
+          }
+        }
+      }
+    }
+  }
+}
+
+html[dir="rtl"] .ft-social-list:first-child  {
+  border-right: 0;
+  border-left: 1px solid var(--primary-blue-80);
+}
+
+/* Footer Main */
+
+.ft-main {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  justify-content: space-between;
+  padding: 3rem 0;
+  
+  & .ft-main-item { /* Items (3 x 1) */
+    padding-right: 1.25rem;
+    min-width: 20%;
+    
+    & a {
+      display: inline-block;
+      font-size: 16px;
+      font-weight: 400;
+      line-height: 22.4px;
+      padding: .7rem;
+      margin: 1rem 0 0;
+      text-decoration: none;
+      color: var(--color-white);
+    }
+
+    & p {
+      font-size: 26px;
+      color: var(--color-white);
+      text-transform: uppercase;
+      font-weight: 700;
+      line-height: 36.4px;
+      margin-top: 0;
+    
+      & span {
+        font-weight: 400;
+      }
+    }
+
+    &:last-child  { /* Footer Main : Newsletter  */ 
+      border-left: 1px solid var(--primary-blue-80);
+      border-right: 0;    
+      margin: 0;
+      min-width: 40%;
+      padding: 0 0 0 5rem;
+      
+      & ul {
+        padding: 0;
+        margin: 0;
+      }
+
+      & a {
+        background-color: var(--color-white);
+        font-size: 14px;
+        line-height: 19.7px;
+        font-weight: 700;
+        color: var(--primary-blue-103);
+        cursor: pointer;
+        border-radius: 6.25rem;
+        border: 0;
+        padding: 1.125rem;
+        width: 7rem;
+        margin: 0;
+      }
+    }
+  }
+}
+
+html[dir="rtl"] .ft-main-item:last-child  {
+  border-left: 0;
+  border-right: 1px solid var(--primary-blue-80);
+  padding: 0;
+}
+
+html[dir="rtl"] .ft-main-item:not(:last-child) {
+  padding-right: 0;
+}
+
+/* Footer Bottom : Legal */
+
+.ft-legal {
+  display: grid;
+  grid-template-columns: 1fr 1.5fr;
+  justify-content: space-between;
+  border-top: 1px var(--primary-blue-80) solid;
+  padding: 1.5rem 0 2rem;
   margin: auto;
+  font-size: 14px;
+  line-height: 16.8px;
+  font-weight: 400;
+  
+  .ft-legal-list {
+    & p {
+      margin: 0;
+      padding: 0;
+    }
+    
+    &:last-child {
+      & ul {
+        display: flex;
+        flex-wrap: nowrap;
+        justify-content: space-between;
+        
+        & a {
+          text-decoration: none;
+          color: var(--color-white);
+        }
+      }
+    }
+  }
+}
+
+/* Tablet */
+
+@media only screen and (width <= 1024px) {
```

Comment:
Implement mobile first approach, write global rules for mobile and override for tab.
Also tab view is <1024px not <=1024